parser_methods.py

```diff
@@ -12,36 +12,35 @@
 # the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
 # either express or implied. See the License for the specific language governing permissions
 # and limitations under the License.
-import sys
-import re
 import csv
+import re
```

Comment:
Similar to other files, the change here is all refactoring from running `pre-commit run --all-files`. 
I have to manually update the too-long lines though.